Expensify/App
# Only load the reports in the LHN

**Fixes (or at least makes somewhat better):** https://github.com/Expensify/Expensify/issues/141692

This is another small step towards improving performance without killing chat switching times.

Rather than load all known pinned and chat reports we'll just load those in the LHN.

cc @rafecolton (if you are up for testing especially with a real device)

**Tests:**
1. Delete existing app and boot up app on iOS
1. Verify you are able to use the app normally
1. Send a message from a report shared with the currently logged in user that was previously read
1. Verify it appears in the LHN and you are able to click on the report and view it's history
